nodes/ovs/ovs.go

```diff
@@ -34,14 +35,31 @@ func (l *ovs) Config() *types.NodeConfig { return l.cfg }
 
 func (l *ovs) PreDeploy(configName, labCADir, labCARoot string) error { return nil }
 
-func (l *ovs) Deploy(ctx context.Context, r runtime.ContainerRuntime) error { return nil }
+func (l *ovs) Deploy(ctx context.Context) error { return nil }
 
-func (l *ovs) PostDeploy(ctx context.Context, r runtime.ContainerRuntime, ns map[string]nodes.Node) error {
+func (l *ovs) PostDeploy(ctx context.Context, ns map[string]nodes.Node) error {
 	return nil
 }
 
 func (l *ovs) WithMgmtNet(*types.MgmtNet) {}
+func (s *ovs) WithRuntime(globalRuntime string, allRuntimes map[string]runtime.ContainerRuntime) {
+	s.runtime = allRuntimes[globalRuntime]
+}
+func (s *ovs) GetRuntime() runtime.ContainerRuntime { return s.runtime }
+
+func (s *ovs) GetContainer(ctx context.Context) (*types.GenericContainer, error) {
+	return nil, nil
+}
 
-func (s *ovs) SaveConfig(ctx context.Context, r runtime.ContainerRuntime) error {
+func (s *ovs) Delete(ctx context.Context) error {
+	return nil
+}
+
+func (s *ovs) GetImages() map[string]string {
+	images := make(map[string]string)
+	images[nodes.ImageKey] = s.cfg.Image
+	return images
+}
```

Comment:
ovs bridge doesn't have an image associated with it

Reply:
fixed.